lib/models/blueprint.js

```diff
@@ -535,14 +535,18 @@ let Blueprint = CoreObject.extend({
       const { Preprocessor } = require('content-tag');
       const preprocessor = new Preprocessor();
       // Strip template tags
-      const templateTagIdentifier = (index) => `template = __TEMPLATE_TAG_${index}__;`;
-      const templateTagIdentifierBraces = (index) => `(template = __TEMPLATE_TAG_${index}__);`;
+      const replacementClassMember = (i) => `template = __TEMPLATE_TAG_${i}__;`;
+      const replacementExpression = (i) => `__TEMPLATE_TAG_${i}__`;
       const templateTagMatches = preprocessor.parse(code);
       let strippedCode = code;
       for (let i = 0; i < templateTagMatches.length; i++) {
         const match = templateTagMatches[i];
         const templateTag = substringBytes(code, match.range.start, match.range.end);
```

Comment:
you may also be interested in content-tag-utils, which does the byte math for you: https://github.com/nullVoxPopuli/content-tag-utils